src/locations/locations.php

```diff
@@ -0,0 +1,6 @@
+<?php
+# When you create a location detail processor add it to this list
+# Files get named as preg_replace("/[^a-z0-9]/", "_", strtolower($_POST['location']['name']));
```

Comment:
we're doing an ID lookup now no more string manipulation so this command is outdated

Reply:
Sort of - this is more to remind me/whoever comes next to follow a convention. But yes, there is no penalty as long as they make the mapping correctly in the array.

Reply:
Updated the comment to reflect that.